hugo/assets/scss/components/teaser.scss

```diff
@@ -55,19 +54,102 @@
             @include sr-only;
         }
 
-        &:focus,
-        &:hover {
-            background: transparentize($c-blue, 0.94);
-        }
-
         &:focus-visible {
             border-color: var(--teaser-border-focus, $c-blue);
         }
     }
 
+    &:not(#{ $self }--blog) {
```

Comment:
the not selector is very specific so I want to avoid using it because it's almost impossible to overrule this line now. Can't we leave the default hover as is and overrule it withing the blog modifier?

Reply:
yes, agreed

Reply:
there was also a transition missing on the teaser__link, therefore not working, so I put the transition on teaser__link instead of the .teaser, so this is also fixed now.